reddit_experiments/__init__.py

```diff
@@ -23,6 +24,7 @@
 
 
 logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
```

Comment:
This is done by `baseplate-serve`: https://github.com/reddit/baseplate.py/blob/develop/baseplate/server/__init__.py#L114

Generally, we shouldn't be setting global logging configuration from libraries -- applications should be the ones doing that. (baseplate-serve being an application though it's in the codebase of a library!)